### ISSUE TITLE
racer error with weird path

### ISSUE THREAD

**Issue by gpaulu** [rust-lang-nursery/rls-vscode#405]
> ERROR 2018-08-23T02:57:04Z: racer::cargo: ERROR!: Os { code: 3, kind: NotFound, message: "The system cannot find the path specified." } C:\Users\appveyor\.cargo\registry\src\github.com-1ecc6299db9ec823\racer-2.0.14\src/racer\cargo.rs 425

I see this in the vscode output section. My username is not "appveyor" :)

Also, when I first installed the extension, it could not find rustup. Rustup is in my PATH. I hardcoded the path to rustup.exe in the settings, and it was able to find it.

I also turned on the following settings but they don't appear to do anything...
```json
"rust-client.revealOutputChannelOn": "info",
"rust-client.logToFile": true
```

I'm using the new [User setup for Windows](https://code.visualstudio.com/updates/v1_26#_user-setup-for-windows) on Windows 10.

**Comment by nrc** [comment]
cc @kngwu

**Comment by kngwyu** [comment]
Please use newer version of rls with racer ^2.1 :pray:

**Comment by jens1o** [comment]
@kngwyu https://github.com/rust-lang-nursery/rls/blob/e67fd48f1aa2a31daee70dc3e80621077a31c52d/Cargo.toml#L27

It is already using `2.1`?

**Comment by kngwyu** [comment]
Yeah current master uses 2.1.4 but I guess @gpaulu uses older(in stable toolchain?) rls with racer 2.0.14 from log messages.

**Comment by gpaulu** [comment]
I'll try that when I get a chance this evening, but the extension installed RLS (and racer?) for me. I'm using stable rustc.

**Comment by nrc** [comment]
>  I'm using stable rustc.

You'll need to switch to nightly Rust to get nightly RLS. However, I think you can set the channel as a user setting in VSCode.
